src/openrct2/network/NetworkPacket.h

```diff
@@ -27,16 +27,19 @@ static_assert(sizeof(PacketHeader) == 6);
 
 struct NetworkPacket final
 {
-    NetworkPacket() = default;
-    NetworkPacket(NetworkCommand id);
+    NetworkPacket() noexcept = default;
+    NetworkPacket(NetworkCommand id) noexcept;
 
-    uint8_t* GetData();
-    const uint8_t* GetData() const;
+    uint8_t* GetData() noexcept;
+    const uint8_t* GetData() const noexcept;
 
-    NetworkCommand GetCommand() const;
+    constexpr NetworkCommand GetCommand() const noexcept
```

Comment:
Also can't be constexpr, should be also back in the cpp file.

Reply:
Reverted constexpr